### ISSUE TITLE
wxwidgets/3.1.4: failed when option `shared=true`

### ISSUE THREAD

**Issue by playgithub** [bincrafters/community#1397]
Error Message
```
..\source_subfolder\src\common\imagjpeg.cpp(399): error C2491: 'wx_jpeg_io_dest': definition of dllimport function not allowed
```

profile
```
[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=Visual Studio
compiler.version=16
build_type=Release
[options]
*:shared=True
[build_requires]
[env]
```

**Comment by ericLemanissier** [comment]
@playgithub this configuration just passed successfully on https://dev.azure.com/bincrafters/packages/_build/results?buildId=8834&view=logs&j=ffd2964e-e0d4-55fb-1c50-31b0e9c92a3e&t=985d871a-f864-559c-fa1d-fca6369a8ccb&l=7128. Could you please give us the command you are running and the full console log ?

**Comment by ericLemanissier** [comment]
the same error is reported on https://trac.wxwidgets.org/ticket/19212#comment:5
it could be solved by wxwidgets 3.1.5 https://github.com/bincrafters/community/pull/1401
